editor/editor_help.cpp

```diff
@@ -4813,6 +4813,7 @@ void FindBar::popup_search() {
 		search_text->select_all();
 		search_text->set_caret_column(search_text->get_text().length());
 		if (grabbed_focus) {
+			results_count_to_current = 0;
```

Comment:
```suggestion
			rich_text_label->deselect();
			results_count_to_current = 0;
```
This isn't always correct, since if you search for something and then remove focus from the search bar without clicking in the EditorHelp (by using the method overview, or clicking the same item in the class list) then using `Ctrl+F` or the `Find...` option to search, it will move the selection to the next match without incrementing the match count.

But if you also deselect first, then it will be forced to search from the start so it would be accurate. There are probably other ways to handle it too, if we don't want to force search from start.

Reply:
Oh, I see now. I don't think I want to change previous behavior. 
WDYT, will it be OK as alternative to reset counter only if `FindBar` was invisible before showing?

Reply:
Suggestion applied, thanks!